x-pack/plugin/core/src/main/java/org/elasticsearch/xpack/core/security/authz/privilege/ClusterPrivilegeResolver.java

```diff
@@ -160,11 +160,12 @@ public static NamedClusterPrivilege resolve(String name) {
         if (fixedPrivilege != null) {
             return fixedPrivilege;
         }
-        logger.error("failed to resolve cluster privilege [" + name + "].");
-        throw new IllegalArgumentException("unknown cluster privilege [" + name + "]. a privilege must be either " +
+        String errorMessage = "unknown cluster privilege [" + name + "]. a privilege must be either " +
             "one of the predefined cluster privilege names [" +
             Strings.collectionToCommaDelimitedString(VALUES.keySet()) + "] or a pattern over one of the available " +
-            "cluster actions");
+            "cluster actions";
+        logger.error(errorMessage);
```

Comment:
I don't think this should be an error.
We would typically not log _anything_ for problems that are also reported in the Rest Response.
If there's a strong argument for doing so here, then it should be `warning` or lower, but I would suggest it's just a debug - the server admin doesn't need to care about this, the person submiting the request does.